src/sidebar/templates/group_list.html

```diff
@@ -15,8 +21,11 @@
         role="button"
         ng-switch on="groups.focused().public"
         title="Change the selected group">
-    <i class="group-list-label__icon h-icon-public" ng-switch-when="true"></i><!-- nospace
-    !--><i class="group-list-label__icon h-icon-group" ng-switch-default></i>
+    <img class="group-list-label__icon group-list-label__icon--third-party"
+         ng-src="{{ thirdPartyGroupIcon }}"
+         ng-if="thirdPartyGroupIcon"
+         ng-switch-when="true"><!-- nospace
+    !--><i class="group-list-label__icon h-icon-public" ng-switch-when="true" ng-if="!thirdPartyGroupIcon"></i><!-- nospace !--><i class="group-list-label__icon h-icon-group" ng-switch-default></i>
```

Comment:
Lines this long are difficult to read in templates. Can you try to wrap them so they are max ~100 chars?